routes/test.js

```diff
@@ -0,0 +1,21 @@
+module.exports.dummy = {
+  path: '/test',
+  method: 'GET',
+  handler: (request, reply) => {
+    reply(`
```

Comment:
can we switch this to use vision with handlebars.  We'll need this later anyways